rubocop-shopify.gemspec

```
@@ -24,4 +24,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 3.1.0"
 
   s.add_dependency("rubocop", "~> 1.62")
+  s.add_dependency("rubocop-thread_safety")
```

Once https://github.com/rubocop/rubocop-thread_safety/pull/84 is released (presumably as `0.7.1`), we'll want

```suggestion
  s.add_dependency("rubocop-thread_safety", ">= 0.7.1")
```